bundle/src/main/resources/SLING-INF/libs/sling/pipes-client/input.html

```diff
@@ -1,8 +1,8 @@
 <div class="col-sm-12 col-md-12">
 	<div class="input-group">
-	  <input type="text" class="form-control" placeholder="Type piped expression...">
+	  <input type="text" class="form-control inputFieldCrea" placeholder="Type piped expression...">
```

Comment:
why not `creation` ? i mean we know we are on an input field, and `Crea` itself is half info